lib/core/CBoostJsonUnbufferedIStreamWrapper.cc

```diff
@@ -9,17 +9,17 @@
  * limitation.
  */
 
-#include <core/CRapidJsonUnbufferedIStreamWrapper.h>
+#include <core/CBoostJsonUnbufferedIStreamWrapper.h>
 
 namespace ml {
 namespace core {
 
-CRapidJsonUnbufferedIStreamWrapper::CRapidJsonUnbufferedIStreamWrapper(std::istream& strm)
+CBoostJsonUnbufferedIStreamWrapper::CBoostJsonUnbufferedIStreamWrapper(std::istream& strm)
     : m_Stream{strm} {};
 
-char CRapidJsonUnbufferedIStreamWrapper::Take() {
+char CBoostJsonUnbufferedIStreamWrapper::Take() {
```

Comment:
Another place where method names could be updated to comply with the coding standards in a followup.